src/playground/sample_data_tooling/sample_data_generator/README.md

```diff
@@ -0,0 +1,187 @@
+# Sample Data Tooling: Generation
+
+Before using this tool, make sure the required libraries as listed in `requirements.txt` are downloaded. As this tool only generates data entries based on the [Faker library](https://faker.readthedocs.io), running this tool alone does not require configuration of OpenSearch (OS) or OpenSearch Dashboards (OSD). 
+
+Additionally, this tool requires some familiarization of Faker methods and optional arguments to both use this tool and find all supported data types.
+
+## Methods
+
+- `choose_field()`: Given a data type, this function will generate a single data value using Faker
+    - **Arguments:**
+        - `kind` (string): The type of variable to be generated
+            - *ex:* `choose_field("ascii_company_email")` taken directly from [ascii_company_email](https://faker.readthedocs.io/en/master/providers/faker.providers.internet.html#faker.providers.internet.Provider.ascii_company_email)
+            ~~~
+            >>> choose_field("ascii_company_email")
```

Comment:
what's the usage of >>> here, I didn't see that parse correctly the markdown.

Reply:
This code block is the Python shell, and that is why there are `...` and `>>>` in the code block.

Reply:
if the code block is Python shell, is there something correct? I'm seeing the code block as following, is it correct format for code block ?

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/33105471/180875133-237f0d75-7a41-46e9-b57a-09e39d4248af.png">

Reply:
I'm assuming that you are using https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Reply:
Yes, the code block is essentially what would be typed on the Shell and outputted. As for the markdown formatting, I did use a fenced code block.

Reply:
you are using ~~~, it should be ```

Reply:
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/33105471/180876962-223a9780-c4d2-474c-b3f1-7b70d6f823b4.png">

Reply:
Done, switched from `~~~` to ```